turbo.json

```diff
@@ -48,7 +48,7 @@
       "persistent": true,
       "dependsOn": ["build"]
     },
-    "check-deduped-deps": {
+    "//#check-deduped-deps": {
```

Comment:
no need for this

Reply:
It runs as a root task, so you have to specify that, correct?